storage/remote/client.go

```diff
@@ -55,7 +55,7 @@ type ClientConfig struct {
 
 // NewClient creates a new Client.
 func NewClient(index int, conf *ClientConfig) (*Client, error) {
-	httpClient, err := httputil.NewClientFromConfig(conf.HTTPClientConfig, "remote_storage")
+	httpClient, err := httputil.NewClientFromConfigAndOptions(conf.HTTPClientConfig, "remote_storage", false)
```

Comment:
This change shouldn't be required in the 2.0 branch, as all http connections use keepalive in 2.0.

Reply:
Good point, will remove.